### PR TITLE
Replace newline by a space

### PR DESCRIPTION
**Pull request by amirbawab**
Closes: #1228 

Simply replaced the newline by a space

### CODE REVIEW

**Comment by sbc100** [comment]
I guess there are no effected tests?

**Comment by amirbawab** [comment]
Looks like the first time it failed for timeout which already has been reported. Second time (empty commit) it passed.

**Comment by sbc100** [comment]
What I mean is it looks like we are missing test coverage, if you can make this change without effecting any tests..

**Comment by amirbawab** [comment]
@sbc100 Added a test case using the original issues' example.

File: test/desugar/ref-func.txt [hunk 1]
```diff
@@ -0,0 +1,14 @@
+;;; TOOL: wat-desugar
```
Comment (sbc100) [inline test/desugar/ref-func.txt]: Thanks!

I think maybe this should live in test/roundtrip/reference-types.txt?

lgtm with that change.

Comment (amirbawab) [reply test/desugar/ref-func.txt]: Updated :+1:

**Pull request merged**